modules/webbrowser/webhelper/CMakeLists.txt

```diff
@@ -24,67 +26,121 @@ set(WEBBROWSER_HELPER_SOURCES
-)
-target_include_directories(${CEF_HELPER_TARGET} PRIVATE 
-	${CMAKE_CURRENT_SOURCE_DIR}
-	"${CEF_INCLUDE_PATH}"
-)
-
-# Use inviwo preprocessor definitions  
-ivw_define_standard_definitions(WebBrowser ${CEF_HELPER_TARGET})
-ivw_define_standard_properties(${CEF_HELPER_TARGET})
-ivw_suppress_compiler_warnings(${CEF_HELPER_TARGET})
-target_compile_definitions(${CEF_HELPER_TARGET} PRIVATE
-    $<$<BOOL:${USE_SANDBOX}>:CEF_USE_SANDBOX>
-)
-
-# Put in ext folder in developer IDE
-ivw_folder(${CEF_HELPER_TARGET} ext/CEF)
+    # Use inviwo preprocessor definitions
+    ivw_define_standard_definitions(WebBrowser ${CEF_HELPER_TARGET})
+    ivw_define_standard_properties(${CEF_HELPER_TARGET})
+    ivw_suppress_compiler_warnings(${CEF_HELPER_TARGET})
+    target_compile_definitions(${CEF_HELPER_TARGET} PRIVATE
+        $<$<BOOL:${USE_SANDBOX}>:CEF_USE_SANDBOX>
+    )
 
+    # Put in ext folder in developer IDE
+    ivw_folder(${CEF_HELPER_TARGET} ext/CEF)
+endif()
 if (OS_MACOSX)
     # Output paths for the app bundles: bin/configuration, for example bin/Debug.
     # Note: This is hardcoded to inviwo app.
     # Other apps need to copy the helper and the cef framework to its framework dir,
     # i.e. add the two commands add_custom_command and install below.
     set(CEF_APP "$<TARGET_BUNDLE_DIR:${CEF_HELPER_TARGET}>/../${IVW_APP_INSTALL_NAME}.app")
-    set(CEF_HELPER_APP "$<TARGET_BUNDLE_DIR:${CEF_HELPER_TARGET}>") # full/path/to/helper.app
-
-    # Copy files into the bin directory (used during development)
-    add_custom_command(
-        TARGET ${CEF_HELPER_TARGET}
-        POST_BUILD
-        # Copy the CEF framework into the Frameworks directory.
-        COMMAND ${CMAKE_COMMAND} -E copy_directory
-        "${CEF_BINARY_DIR}/Chromium Embedded Framework.framework"
-        "${CEF_APP}/Contents/Frameworks/Chromium Embedded Framework.framework"
-        COMMAND ${CMAKE_COMMAND} -E copy_directory
-        "${CEF_HELPER_APP}"
-        "${CEF_APP}/Contents/Frameworks/${CEF_HELPER_OUTPUT_NAME}.app"
-        VERBATIM
-    )
-    # Copy the helper app bundle into the Frameworks directory (used by installer)
-    install(DIRECTORY "${CEF_HELPER_APP}"
-            DESTINATION "${IVW_APP_INSTALL_NAME}.app/Contents/Frameworks/${CEF_HELPER_OUTPUT_NAME}.app"
-            COMPONENT inviwo-module-webbrowser)
 
-    set_target_properties(${CEF_HELPER_TARGET} PROPERTIES
-        MACOSX_BUNDLE_INFO_PLIST ${CMAKE_CURRENT_SOURCE_DIR}/mac/helper-Info.plist
-    )
+    set(CEF_TARGET inviwo-module-webbrowser)
     if(USE_SANDBOX)
         # Logical target used to link the cef_sandbox library.
         ADD_LOGICAL_TARGET("cef_sandbox_lib" "${CEF_SANDBOX_LIB_DEBUG}" "${CEF_SANDBOX_LIB_RELEASE}")
-        target_link_libraries(${CEF_HELPER_TARGET} PRIVATE cef_sandbox_lib)
+        #target_link_libraries(${CEF_HELPER_TARGET} PRIVATE cef_sandbox_lib)
     endif()
+    # Create the multiple Helper app bundle targets.
+    foreach(_suffix_list ${CEF_HELPER_APP_SUFFIXES})
```

Comment:
why all the prefix "_"?

Reply:
Taken from the CEF example. They now use three different executables for the helpers...